src/edu/stanford/dlss/was/WasapiFile.java

```diff
@@ -25,6 +25,9 @@
   @JsonProperty("crawl-start")
   private String crawlStartDateStr;
 
+  @JsonProperty("crawl-time")
+  private String crawlTimeDateStr;
```

Comment:
I believe we were using the `DateStr` suffix to clarify what `crawl-start` meant;  would it be clearer to have this be `fileCrawlDateStr` or something like that?

"Time the original content of the file was crawled" is the definition per https://github.com/WASAPI-Community/data-transfer-apis/blob/master/ait-implementation/wasapi/implemented-swagger.yaml#L177-L180